.github/workflows/dependabot-batch-updater.yml

```
@@ -0,0 +1,19 @@
+name: Check if dependencies can be updated and create a pull request with updated ones.
+on:
+  schedule:
+    # Run once a week at 5:00
+    - cron:  '0 5 * * */7'
+
+jobs:
+  update_dependencies:
+    runs-on: ubuntu-20.04
+    name: Update dependencies
+    steps:
+      - uses: rasaHQ/dependabot-batch-updater@main
+        with:
+          username: ""
+          repo-token: ${{ secrets.DEPENDABOT_UPDATER_GITHUB_TOKEN || secrets.GITHUB_TOKEN}}
```

do we need to create a new secret or is it already available? What kind of permissions does it need?

---

Let's use `RASABOT_GITHUB_TOKEN`:
```suggestion
          repo-token: ${{ secrets.RASABOT_GITHUB_TOKEN }}
```